README.md

```diff
@@ -299,6 +295,49 @@ to setup Politeia and access it through the UI.
 
 ### Further information
 
+
+#### politeiawww user databaes options
+
+Both Pi and CMS use the same politeiawww user database.  The default user
+database is LevelDB, a simple key-value store.  This is fine if you're just
+getting started, but LevelDB has some scalability limitations due to it being a
+simple key-value store that doesn't allow concurrent connections.
+
+A more scalable option is setting up the user database to use CockroachDB.  The
+CockroachDB implementation makes public user fields queryable and encrypts
+private user data at rest.  You can setup the user database to use CockroachDB
+with the following commands.  Before running these commands, make sure that
+you've followed the instructions above and have a CockroachDB instance running.
+
+Create a CockroachDB user database and assign user privileges:
+
+    cd $GOPATH/src/github.com/decred/politeia
+    ./scripts/userdbsetup.sh
+
+Create an encryption key to be used to encrypt data at rest:
+
+    $ politeiawww_dbutil -createkey
+    Encryption key saved to: ~/.politeiawww/dbkey.json
+
+Add the following settings to your politeiawww config file.  The encryption key
+location may be different depending on your operating system.
+
+    userdb=cockroachdb
+    encryptionkey=~/.politeiawww/dbkey.json
+
+##### Rotating encryption key
+
+Encryption keys can be rotated using the `oldencryptionkey` politeiawww config
+setting.  To rotate keys, set `oldencryptionkey` to the existing key and set
+`encryptionkey` to the new key.  Starting politeiawww with both of these config
+params set will kick off a key rotation.
+
```

Comment:
Neat but does this re-encrypt the entire db? I ask because what happens if you do this twice?

Reply:
Yes, it will re-encrypt the entire db.  If you do it twice, without changing the config params, it will throw a `cannot decrypt` error on the first user it tries to decrypt because the old key is no longer valid.